src/bramble/client/ProjectStats.js

```diff
@@ -0,0 +1,263 @@
+/* eslint
```

Comment:
It goes back to the fact that you are not doing any style checking with you eslint configurations